target/check_for_changes.sh

```diff
@@ -0,0 +1,124 @@
+		sed -i -e '/\!include auth-ldap\.conf\.ext/s/^/#/' /etc/dovecot/conf.d/10-auth.conf
+		sed -i -e '/\!include auth-passwdfile\.inc/s/^#//' /etc/dovecot/conf.d/10-auth.conf
+		# Creating users
+		# 'pass' is encrypted
+		# comments and empty lines are ignored
+		grep -v "^\s*$\|^\s*\#" /tmp/docker-mailserver/postfix-accounts.cf | while IFS=$'|' read login pass
+		do
+			# Setting variables for better readability
+			user=$(echo ${login} | cut -d @ -f1)
+			domain=$(echo ${login} | cut -d @ -f2)
+			# Let's go!
+			echo "${login} ${domain}/${user}/" >> /etc/postfix/vmailbox
+			# User database for dovecot has the following format:
+			# user:password:uid:gid:(gecos):home:(shell):extra_fields
+			# Example :
+			# ${login}:${pass}:5000:5000::/var/mail/${domain}/${user}::userdb_mail=maildir:/var/mail/${domain}/${user}
+			echo "${login}:${pass}:5000:5000::/var/mail/${domain}/${user}::" >> /etc/dovecot/userdb
+			mkdir -p /var/mail/${domain}
+			if [ ! -d "/var/mail/${domain}/${user}" ]; then
+				maildirmake.dovecot "/var/mail/${domain}/${user}"
+				maildirmake.dovecot "/var/mail/${domain}/${user}/.Sent"
+				maildirmake.dovecot "/var/mail/${domain}/${user}/.Trash"
+				maildirmake.dovecot "/var/mail/${domain}/${user}/.Drafts"
+				echo -e "INBOX\nSent\nTrash\nDrafts" >> "/var/mail/${domain}/${user}/subscriptions"
+				touch "/var/mail/${domain}/${user}/.Sent/maildirfolder"
+			fi
+			# Copy user provided sieve file, if present
+			test -e /tmp/docker-mailserver/${login}.dovecot.sieve && cp /tmp/docker-mailserver/${login}.dovecot.sieve /var/mail/${domain}/${user}/.dovecot.sieve
+			echo ${domain} >> /tmp/vhost.tmp
+		done
+	fi
+    # regen postfix aliases
+    echo -n > /etc/postfix/virtual
+	echo -n > /etc/postfix/regexp
+	if [ -f /tmp/docker-mailserver/postfix-virtual.cf ]; then
+		# Copying virtual file
+		cp -f /tmp/docker-mailserver/postfix-virtual.cf /etc/postfix/virtual
+		while read from to
+		do
+			# Setting variables for better readability
+			uname=$(echo ${from} | cut -d @ -f1)
+			domain=$(echo ${from} | cut -d @ -f2)
+			# if they are equal it means the line looks like: "user1     other@domain.tld"
+			test "$uname" != "$domain" && echo ${domain} >> /tmp/vhost.tmp
+		done < /tmp/docker-mailserver/postfix-virtual.cf
+	fi
+	if [ -f /tmp/docker-mailserver/postfix-regexp.cf ]; then
+		# Copying regexp alias file
+		cp -f /tmp/docker-mailserver/postfix-regexp.cf /etc/postfix/regexp
+		sed -i -e '/^virtual_alias_maps/{
+		s/ regexp:.*//
+		s/$/ regexp:\/etc\/postfix\/regexp/
+		}' /etc/postfix/main.cf
+	fi
+    # Set vhost 
+	if [ -f /tmp/vhost.tmp ]; then
+		cat /tmp/vhost.tmp | sort | uniq > /etc/postfix/vhost && rm /tmp/vhost.tmp
+	fi
+    
+    # Set right new if needed
+	if [ `find /var/mail -maxdepth 3 -a \( \! -user 5000 -o \! -group 5000 \) | grep -c .` != 0 ]; then
+		chown -R 5000:5000 /var/mail
+	fi
+    
+    
+    #supervisorctl restart postfix # Buggy causes error while testing.
+    postfix reload
```

Comment:
This is my only concern because the process might lose the pid. Is the reload nesesary?
When we use the postmap a reload is not needed (http://www.postfix.org/postmap.1.html)